airflow/www/views.py

```diff
@@ -2474,6 +2506,7 @@ def confirm(self):
         dag_run_id = args.get("dag_run_id")
         state = args.get("state")
         origin = get_safe_url(args.get("origin"))
+        group_id = args.get("group_id")
```

Comment:
I do wonder if we need separate argument for this -- task_ids and group_ids can never overlap, so given an ID we should be able to find it in the DAG and work out if it's a task or a group...

Reply:
We follow similar pattern for ClearTask. https://github.com/apache/airflow/blob/main/airflow/www/views.py#L2152
Let me know if this needs to change as well.

Having a single parameter `node_id` makes more sense, but we will have to add logic to find if it is a task or a group. From the views we already know if it is a group or not, can we send node_id and isGroup parameters?